hibernate-core/src/test/java/org/hibernate/jpa/test/persistenceunit/DuplicatePersistenceUnitNameTest.java

```diff
@@ -75,13 +75,13 @@ public void testDuplicatePersistenceUnitNameLogAWarnMessage() {
 
 		public TestClassLoader() {
 			urls = Arrays.asList(
-				findAsResource(
-					"org/hibernate/jpa/test/persistenceunit/META-INF/persistence.xml"
-				)
-				,
-				findAsResource(
-					"org/hibernate/jpa/test/persistenceunit/META-INF/persistenceUnitForNameDuplicationTest.xml"
-				)
+					ClassLoaderServiceTestingImpl.INSTANCE.locateResource(
```

Comment:
Hard to tell from just diff... did you change these from tabs to spaces?  Or just change the tab-indentation-depth?  Either way, this looks wrong

Reply:
this is IntelliJ's auto formatting utility acting up I'm afraid